### ISSUE TITLE
[Error] Rarely getting Meteor in Color

### ISSUE THREAD

**Issue by flewid** [reynico/raspberry-noaa#122]
For some reason in 2 weeks I have only received 2 color Meteor, they are almost always black and white. 

I've tried 100, 144, 248 for the samplerate from that other thread just to see if that made a difference (it did not). 

Here are my details: 

**Location:** 
`Eastern Time Zone `

**GPS according to google maps:** 
```
45.962608
-75.809464

```
**Dongle**
`NooElec SmarTee XTR `

**noaa.conf** 
```
NOAA_HOME=/home/radio/raspberry-noaa
NOAA_OUTPUT=/var/www/wx
METEOR_OUTPUT=/var/www/wx/meteor
RAMFS_AUDIO=/var/ramfs
SAT_MIN_ELEV=25
METEOR_MIN_ELEV=25
SUN_MIN_ELEV=10
LOG_LEVEL=DEBUG
LAT=45.962608
LON=-75.809464
BIAS_TEE="-T"
DELETE_AUDIO="false"
FLIP_METEOR_IMG="true"
GAIN=42.0
SCHEDULE_ISS="true"
```

**predict.qth**
```
BILLY
 45.9626
 75.8095
 223
```

**sun.py**
```
#!/usr/bin/env python3
import ephem
import time
import sys
timezone = -5 + time.localtime().tm_isdst
date = time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(int(sys.argv[1])-(timezone*60*60)))

obs=ephem.Observer()
obs.lat='45.963000'
obs.long='-75.809160'
obs.date = date

sun = ephem.Sun(obs)
sun.compute(obs)
sun_angle = float(sun.alt) * 57.2957795 # Rad to deg
print(int(sun_angle))
```

**Comment by tegwilym** [comment]
You might try lowering your SUN_MIN_ELEV in .noaa.conf to something lower.  I have mine set at 1 and I'm getting at least one daylight image a day from Meteor.   I'm in NE Wisconsin, so I'm low on daylight here too.

**Comment by flewid** [comment]
@tegwilym thanks, will try that

**Comment by flewid** [comment]
thanks @tegwilym setting it to 1 has worked!

**Comment by tegwilym** [comment]
> thanks @tegwilym setting it to 1 has worked!

Great!  I'm still learning the tricks of working out the different things in the configuration files.  Glad I've learned enough to give some tech support!   :-)  

reynico, looks like this can be closed.  I'm not sure where I do that or if you need to do that?  

Thanks, 
Tom

**Comment by klikis** [comment]
mine is set to 10, in fact I get black and white images, do you recommend me to set it to 1?
![METEOR-M220210130-073742-122-rectified](https://user-images.githubusercontent.com/9518512/106350452-c3f37300-62d5-11eb-933f-4c6618fb7f11.jpg)

**Comment by tegwilym** [comment]
> mine is set to 10, in fact I get black and white images, do you recommend me to set it to 1?

Yes, try a lower sun angle.  You can even us a negative number I’ve heard. I have mine set to 1 and I’m getting color on both meteor and NOAA again. 
Here is the Great Lakes area from yesterday morning. 
Tom 

![D768EB84-8365-4A36-A97A-154BA979664D](https://user-images.githubusercontent.com/43358320/106352278-49abfa00-62a7-11eb-95f1-b5132c2ef1dd.jpeg)

**Comment by klikis** [comment]
ok thanks

**Comment by klikis** [comment]
thanks......

![METEOR-M220210202-081808-122-rectified](https://user-images.githubusercontent.com/9518512/106576530-8abd3c00-653d-11eb-8e8f-a2106751964d.jpg)

**Comment by F1SMV** [comment]
your pic is nice..my last réception..:) 
![METEOR-M220210208-192212-122-rectified](https://user-images.githubusercontent.com/64283688/107320379-dec9a280-6aa0-11eb-8654-47f49bcef514.jpg)
